test/mjml-tags.spec.tsx

```diff
@@ -24,18 +23,18 @@ describe("mjml tags", () => {
       const variable = "Nice";
       expect(
         renderToMjml(<tags.MjmlTitle>{variable} Content</tags.MjmlTitle>)
-      ).to.equal("<mj-title>Nice Content</mj-title>");
+      ).toBe("<mj-title>Nice Content</mj-title>");
     });
 
     it("should render functional component", () => {
-      const HelloWorld = () => "Hello World!";
+      const HelloWorld = () => <>Hello World!</>;
```

Comment:
Was throwing a TS error for not being able to render a string, unsure if this disregards the purpose of this test